api/v3/wfs_validation.go

```diff
@@ -1,74 +1,82 @@
 package v3
 
 import (
-	"fmt"
 	"strings"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
 	sharedValidation "github.com/pdok/smooth-operator/pkg/validation"
 )
 
 func (wfs *WFS) ValidateCreate() ([]string, error) {
-	warnings := []string{}
```

Comment:
Dit is een ding waar IDEs over klagen maar wat beter expliciet is. Als je de array niet expliciet instantieert dan is de array nil totdat het eerste element wordt toegevoegd. Dit kan leiden tot NPEs. Ik raad aan om dit weer terug te zetten.